hedera-node/configuration/dev/node.properties

```diff
@@ -1 +1,2 @@
 hedera.profiles.active=DEV
+hedera.recordStream.logDir=data/recordstreams/
```

Comment:
should this have `netty.mode=TEST` added to it

Reply:
I updated so that if `hedera.profiles.active=DEV` then Netty is configured in non-`PROD` mode.